_posts/2023-09-05-Python-Packaging-as-Publishing.md

```diff
@@ -0,0 +1,300 @@
+
+## Why practice packaging?
+
+{% include figure.html image="images/text-vs-book.png" caption="Image comparing how a page with some text and a book look and feel different."  %}
+
+The practice of Python packaging efforts is similar to that of publishing a book. Consider how a bag of text is different from a book. How and _why_ are these things different?
+
+- A book has commonly understood sequencing of content (i.e. copyright page, then title page, then body content pages...).
+- A book often cites references and acknowledges other work explicitly.
+- A book undergoes a manufacturing process which allows the text to be received in many places the same way.
+
+{% include figure.html image="images/python-packaging-to-audience.png" caption="Image showing code undergoing packaging to achieve understanding, trust, and connection for an audience."  %}
+
+These can be thought of metaphors when it comes to packaging in Python. Just like books, software can sometimes have a [code smells](https://en.wikipedia.org/wiki/Code_smell) we want to avoid. Following good packaging practices helps to avoid unwanted code smells while increasing development velocity, maintainability of software through understandability, trustworthiness of the content, and connection to other projects.
+
+Note: these techniques can also work just as well for [inner source](https://en.wikipedia.org/wiki/Inner_source) collaboration (private or proprietary development within organizations)! Don't hesitate to use these on projects which may not be public facing in order to make development and maintenance easier (if only for you).
```

Comment:
You could use the recently added alert component for this, and possibly even the tip of the week blurb at the top of every article.

Reply:
Thanks for this, I've added changes to implement the alert capability for a few body content blocks and also the "intro" blurb at the top.